docker-for-ibm-cloud/faqs.md

```diff
@@ -18,13 +18,16 @@ Docker EE for IBM Cloud is an unmanaged, native Docker environment within IBM Cl
   Make sure that you enable the permissions in the following table.
 
   * The **View Only** user role does not have any of these enabled by default.
-  * The **Basic User** role has some of these enabled by default. Permissions marked with `*` designate new permissions that **Basic Users** need.
-  * The **Super User** role has everything enabled.
+  * The **Basic User** role has some of these enabled by default.
+  * The **Super User** role has most of these enabled by default.
 
   > Save your setting changes!
   >
   > Don't forget to click **Set Permissions** as you go through the tabs of each permission set so that you don't lose your settings.
 
+  {% capture superscript-link1 %}[1](#edge-footnote1){: style="vertical-align: super; font-size: smaller;" }{% endcapture %}
+  {% capture superscript-link2 %}[2](#edge-footnote2){: style="vertical-align: super; font-size: smaller;" }{% endcapture %}
```

Comment:
These snippets are supposed to allow superscripts in the table below. However, the deploy preview doesn't seem to render it properly:

![image](https://user-images.githubusercontent.com/20074219/35443558-857845de-0279-11e8-9dda-43e89ff16438.png)

@joaofnfernandes any pointers? I'm sure I must have formatted it wrong!

Reply:
As we discussed offline, you can use just html to implement this.